content/features/50-runtimes-monitoring.adoc

```diff
@@ -0,0 +1,15 @@
+---
+title: "Runtimes Monitoring"
+date: 2019-03-06T13:55:38+02:00
+draft: false
+subTitle: "Monitor your application metrics within Kiali"
+image: features/runtimes_monitoring.png
+weight: 50
+---
+
+Kiali comes with some default dashboards for a couple of runtimes: Thorntail, Vert.x, Spring Boot and Node.js.
```

Comment:
Same comment as above about alphabetizing the list.   Node.js, Spring Boot, Thorntail, Vert.x.